### ISSUE TITLE
Please add default value for parameter select

### ISSUE THREAD

**Issue by ichbintonywu** [neo4j-labs/neodash#190]
## Guidelines
Please note that GitHub issues are only meant for bug reports/feature requests.

## Feature request template

**Is your feature request related to a problem? Please describe.**
Please add default value for parameter select, so that the table or other format output can have results using the default and preset values, it will be great if we can have such a feature in the next release. 

**Describe the solution you'd like**
Empty input will invoke "Expected parameter (s)"

**Describe alternatives you've considered**
Using case when with default value could not solve this issue.

_with case $neodash_host_id when 
"" then "3c8e4bb1-588d-4179-b58f-ca1c4ed84f9e" else $neodash_host_id end as val

match (n:Host) where n.id =
val
return n.id_

**Additional context**
none

**Comment by nielsdejong** [comment]
Hi @ichbintonywu - this features is added in https://github.com/neo4j-labs/neodash/pull/200.
You should see it in an upcoming release.

Thanks for your suggestion!
Niels

**Comment by nielsdejong** [comment]
Default values are available in v2.1.7 onwards. Thanks for the suggestion,
Niels

**Comment by nukerxy** [comment]
If you use an older version and cannot upgrade, here is a dirty workaround:
Replace dots, comments and `$neodash_node_property` accordingly.

```
MATCH (....)
WHERE $neodash_node_property IS NULL
/* something something */
RETURN ...
UNION
MATCH (....)
WHERE $neodash_node_property IS NOT NULL
/* something something */
RETURN ....;
```
